main.nf

```diff
@@ -1307,39 +1420,223 @@ process IVAR_SNPEFF {
+        "ANN[*].BIOTYPE" "ANN[*].RANK" "ANN[*].HGVS_C" \\
+        "ANN[*].HGVS_P" "ANN[*].CDNA_POS" "ANN[*].CDNA_LEN" \\
+        "ANN[*].CDS_POS" "ANN[*].CDS_LEN" "ANN[*].AA_POS" \\
+        "ANN[*].AA_LEN" "ANN[*].DISTANCE" "EFF[*].EFFECT" \\
+        "EFF[*].FUNCLASS" "EFF[*].CODON" "EFF[*].AA" "EFF[*].AA_LEN" \\
+        > ${prefix}.snpSift.table.txt
+   	"""
 }
 
 /*
- * STEP 5.5.2.3: iVar consensus sequence report with QUAST
+ * STEP 5.6.2.3: iVar consensus sequence report with QUAST
  */
 process IVAR_QUAST {
     label 'process_medium'
-    publishDir "${params.outdir}/variants/ivar", mode: params.publish_dir_mode
+    publishDir "${params.outdir}/variants/ivar/quast", mode: params.publish_dir_mode
 
     when:
     !params.skip_variants && 'ivar' in callers && !params.skip_variants_quast
 
     input:
-    path consensus from ch_ivar_consensus_fasta.collect{ it[2] }
+    path consensus from ch_ivar_consensus.collect{ it[2] }
     path fasta from ch_fasta
     path gff from ch_gff
 
     output:
-    path "quast" into ch_ivar_quast_mqc
+    path "AF${params.max_allele_freq}" into ch_ivar_quast_mqc
 
     script:
     features = params.gff ? "--features $gff" : ""
     """
     quast.py \\
-        --output-dir quast \\
+        --output-dir AF${params.max_allele_freq} \\
         -r $fasta \\
         $features \\
         --threads $task.cpus \\
         ${consensus.join(' ')}
     """
 }
 
+////////////////////////////////////////////////////
+/* --              BCFTOOLS                    -- */
+////////////////////////////////////////////////////
+
+/*
+ * STEP 5.6.3: Variant calling with BCFTools
+ */
+process BCFTOOLS_VARIANTS {
+    tag "$sample"
+    label 'process_medium'
+    publishDir "${params.outdir}/variants/bcftools", mode: params.publish_dir_mode,
+        saveAs: { filename ->
+                      if (filename.endsWith(".txt")) "bcftools_stats/$filename"
+                      else filename
+                }
+
+    when:
+    !params.skip_variants && 'bcftools' in callers
+
+    input:
+    tuple val(sample), val(single_end), path(bam) from ch_markdup_bam_bcftools
+    path fasta from ch_fasta
+
+    output:
+    tuple val(sample), val(single_end), path("*.vcf.gz*") into ch_bcftools_variants_consensus,
+                                                               ch_bcftools_variants_snpeff
+    path "*.bcftools_stats.txt" into ch_bcftools_variants_mqc
+
+    script:
+    """
+    bcftools mpileup \\
+        --count-orphans \\
+        --no-BAQ \\
+        --max-depth 50000 \\
+        --fasta-ref $fasta \\
+        --min-BQ $params.min_base_qual \\
+        --annotate FORMAT/AD,FORMAT/ADF,FORMAT/ADR,FORMAT/DP,FORMAT/SP,INFO/AD,INFO/ADF,INFO/ADR \\
+        ${bam[0]} \\
+        | bcftools call --output-type v --ploidy 1 --keep-alts --keep-masked-ref --multiallelic-caller \\
+        | bcftools view --output-file ${sample}.vcf.gz --output-type z --include 'INFO/DP>=5'
```

Comment:
This isn't the same filter (DP >= 5) as used with the other callers

Reply:
Fixed https://github.com/nf-core/viralrecon/pull/93/commits/a495c1c40bd21085c21247ad812e51bcbb90088f